### ISSUE TITLE
Issue with San Storage in New-OVServerProfileTemplate

### ISSUE THREAD

**Issue by oolivera15** [HewlettPackard/POSH-HPOneView#600]
**Expected Behavior**
New-OVServerProfileTemplate should create the server profile with the SAN Storage information.

**Actual Behavior**
New-OVServerProfileTemplate create the server profile without the SAN Storage information.
This information can be added via the web interface, but it does not work with the API.

**Steps to reproduce**

```
#Attributes for ServerProfileTemplate
$name                               = "template1"
$shtName                            = "DL380p Gen8 1"
$sht                                = Get-OVServerHardwareType -Name $shtName


#Connections section
$cnConsistency                      = "Minimum"

#Attributes for connection
$connID                             = 1
$connName                           = "FC_A"
$connType                           = "FibreChannel"
$netName                            = "SAN_A"
$ThisNetwork                        = Get-OVNetwork -Type FibreChannel -Name $netName
$requestedMbps                      = "Auto"
$Conn1                              = New-OVServerProfileConnection -ConnectionID $connID -Name $connName -ConnectionType $connType -Network $ThisNetwork -RequestedBW $requestedMbps 
$connections                        = $Conn1

#Attributes for SAN Storage
$osType                             = "VMware"

#SAN volume attributes for volume ID "1"
$volId                              = 1
$lunIdType                          = "Auto"
$volName                            = "volume1"
$volume1                            = Get-OVStorageVolume -Name $volName
$volume1                            = New-OVServerProfileAttachVolume -VolumeID $volId -LunIDType $lunIdType -Volume $volume1
$volumeAttachments                  = $volume1
$sanConsistency                     = "Minimum"

New-OVServerProfileTemplate -Name $name -ServerHardwareType $sht -ManageConnections $True -ConnectionsConsistencyChecking $cnConsistency  -Connections $connections -SanStorage -HostOsType $osType  -StorageVolume $volumeAttachments -SANStorageConsistencyChecking $sanConsistency
```

**Version Information**
Powershell:
```
> Get-Host | Select-Object Version

Version
-------
7.0.4
```
HPEOneView: 6.50.00-0452161

HPEOneView module:
```
> (Get-Module -Name HPEOneView.630).Version

Major  Minor  Build  Revision
-----  -----  -----  --------
6      30     2928   3678
```

Could you please help us to solve this issue?
Thank you!

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 6.30.3188.1900](https://github.com/HewlettPackard/POSH-HPEOneView/releases/tag/v6.30.3188.1900) and in [PowerShell Gallery](https://www.powershellgallery.com/packages/HPEOneView.630/6.30.3188.1900).

Please verify before this issue is closed.

**Comment by ChrisLynchHPE** [comment]
Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
